docs/the-basics-errors.md

```diff
@@ -25,20 +25,20 @@ module.exports = async function HandleError(context, props) {
 };
 ```
 
-The example code above will do following things for you while an error occurred:
+The above example does the following things while an error triggered:
 
-1. Send `There are some unexpected errors happened. Please try again later, sorry for the inconvenience.` text message to the end user.
-2. Log error to the console.
-3. Log error into error tracker.
-4. Send `error.stack` as text message to the user (only in the development)
+1. Send a "There are some unexpected errors happened. Please try again later, sorry for the inconvenience." text message to the end user.
+2. Log the error to the console.
+3. Log the error into the error tracker.
+4. Send `error.stack` as a text message to the user (only in development)
 
-> **Note:** In LINE channel, errors should be handled in a different way because reply API can only be called once. You can read more about it in the [Error Handling in LINE documentation](channel-line-errors.md).
+> **Note:** In the LINE channel, you must handle errors differently because the reply token can only be used once. For more information, see [Error Handling in LINE](channel-line-errors.md).
```

Comment:
I think `While building a LINE bot,` instead of `In the LINE channel,` makes more sense to me here